### PR TITLE
[multikueue] Reconcile the relevant ACs on cluster create.

### LINKED ISSUE #2117: It seems like the wrong function is being used when a multiKueueCluster is created
**What happened**:
when a multiKueueCluster is created, the function named **queueReconcileForConfigUsers** is used directly in admissioncheck.go#L270

**What you expected to happen**:
the function named **queue**(admissioncheck.go#L313) should be used, then the function named **queueReconcileForConfigUsers** used in queue function.

**How to reproduce it (as minimally and precisely as possible)**:
when a multiKueueCluster is created

**Anything else we need to know?**:
https://github.com/kubernetes-sigs/kueue/blob/43e5eb8ee19fe69d3dd3f03c13f4ad46b77dd094/pkg/controller/admissionchecks/multikueue/admissioncheck.go#L264-L273

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
[multikueue] Reconcile the relevant ACs on cluster create.

#### Which issue(s) this PR fixes:

Fixes #2117 

#### Does this PR introduce a user-facing change?

```release-note
MultiKueue: MultiKueue: Fix a bug that could potentially cause the failure of MultiKueue creation.
```

### CODE REVIEW

File: pkg/controller/admissionchecks/multikueue/admissioncheck.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func (m *mkClusterHandler) Create(ctx context.Context, event event.CreateEvent,
 		return
 	}
 
-	if err := queueReconcileForConfigUsers(ctx, mkc.Name, m.client, q); err != nil {
+	if err := m.queue(ctx, mkc, q); err != nil {
```
Comment (tenzen-y) [inline pkg/controller/admissionchecks/multikueue/admissioncheck.go:270]: Doesn't the fact that we could not detect this bug in the test indicate that we don't have any cases to verify here?

Comment (trasc) [reply pkg/controller/admissionchecks/multikueue/admissioncheck.go:270]: The problem is that we don't test (and I don't think it's easy to do) the create only scenario, after the cluster is created one of the first things happening is for it's active condition to be set, triggering an Update which worked well.

Comment (tenzen-y) [reply pkg/controller/admissionchecks/multikueue/admissioncheck.go:270]: Uhm, couldn't add a unit test for creation?

Comment (trasc) [reply pkg/controller/admissionchecks/multikueue/admissioncheck.go:270]: That will be the first if it's king in the entire project.

Comment (tenzen-y) [reply pkg/controller/admissionchecks/multikueue/admissioncheck.go:270]: > That will be the first if it's king in the entire project.

@trasc Alright. Could you create tracking issue?

/lgtm
/approve

**Comment by tenzen-y** [comment]
```release-note
MultiKueue: Fix a bug that could potentially cause the failure of MultiKueue creation.
```

**Comment by tenzen-y** [comment]
/release-note-edit

```release-note
MultiKueue: Fix a bug that could potentially cause the failure of MultiKueue creation.
```

**Pull request merged**